treeherder/seta/analyze_failures.py

```diff
@@ -94,14 +110,8 @@ def get_failures_fixed_by_commit():
         # 40char revision we will have two disjunct set of failures
         #
         # Some of this will be improved in https://bugzilla.mozilla.org/show_bug.cgi?id=1323536
-        if revision_id not in failures:
-            failures[revision_id] = []
```

Comment:
don't we need to define failures[revision_id] so we can append to it a few lines down (line 136)